src/utils/gl_quadrature.cpp

```diff
@@ -1,1458 +0,0 @@
-//========================================================================================
-// Athena++ astrophysical MHD code
-// Copyright(C) 2014 James M. Stone <jmstone@princeton.edu> and other code contributors
-// Licensed under the 3-clause BSD License, see LICENSE file for details
```

Comment:
I guess this quadrature is all from Athena++ and is not used? Should we really get rid of it? As someone who added quadrature routines to another code a couple weeks ago, it seems very possible this could prove useful to someone in the future and it is pretty clearly utility code so it doesn't make the repo much more confusing.

Reply:
That's a fair point---I agree quadrature routines are generically useful. My thinking was that probably this code will need to be changed pretty substantially to be performance portable.